visualize-admin/visualization-tool
# Fix/smaller improvements

**This PR:** 
- Adds Feedback to the User Account Dropdown
- This PR resolves #1802 
- This PR resolves #1828 
- (These PR's are practically the same)

**Details**
@bprusinowski not sure how to better position the nested `Menu` let me know about potential improvements

---

### How to Test 
1. This can only be tested once deployed
2. Login
3. Open the Dropdown See the new `Feedback >` button ✅
4. Click the `Feedback >` button see how there is Report Bug and
